service-csw/src/main/java/fi/nls/oskari/csw/domain/CSWIsoRecord.java

```diff
@@ -695,8 +704,11 @@ public JSONObject toJSON() {
             private Date date;
             private String dateType;
             private String xmlDate;
+            private static final SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd", Locale.US);
```

Comment:
SimpleDateFormat is not threadsafe, but you could do:

     private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd");

And layer on in setXmlDate():

         date.toInstant().format(DATE_FORMATTER);